### ISSUE TITLE
quiz display issue

### ISSUE THREAD

**Issue by cacase** [Alex-Devonte/PHP-Fall-2017#1]
I uploaded the files and made a test database for the quiz.  When I went to display the quiz it gave me the first quiz answer as the question. I made a change to the numbers inside the array on line 34 and 44 of quiz.php to adjust this.  (Decreased the number by 1 on each of the $row[]).

I made a few small changes to the login.php file. The array on line 23 I changed for 6 to 5 to match with email on the table quizusers . On line 13 at the end of $statement =  I changed the question mark to = 'userid'. (I was getting an Undefined variable error.)  I did add a header and spaces but that is just cosmetic and may be changed when the css is added.

I am trying to figure out how to do a pull request and upload the file for comparison.  Still learning how to use github. I apologize if I get it wrong. 
[quizAndLoginfile.zip](https://github.com/Alex-Devonte/PHP-Fall-2017/files/1524040/quizAndLoginfile.zip)

**Comment by Alex-Devonte** [comment]
When making your tables, did you include and id field for each?
Prettyman said each table should have an ID so I made my database tables with that in mind which is why the array seemed to be one index more than it should.

With the ID field in the table, if you start from zero and count each field the numbers in my arrays should match up.

**Comment by cacase** [comment]
I did not add that for my table, but I did wonder if that was what you had done.  I post the change if you had not. I should have put that in my message.

**Comment by Alex-Devonte** [comment]
Yeah, I was having trouble with it myself until I realized I had added it.
